src/cmd/wallet.rs

```diff
@@ -270,35 +272,57 @@ impl SpendOpt {
             (psbt, vec![])
         };
 
-        party.wallet().sign(&mut psbt, SignOptions::default())?;
```

Comment:
We no longer care about `finalized`, but instead get different CmdOutput `output` based on each branch (currently coldcard or not).